### PR TITLE
Issue/3851 update classifiers in setup py

### LINKED ISSUE #3851: Update classifiers in setup.py
The classifiers in setup.py still list python 3.5 on master, while 3.8 and 3.9 are not mentioned at all:
https://github.com/inmanta/inmanta-core/blob/b364df85f4e90b5fde983c1721f02205e7090edb/setup.py#L65

### PR DESCRIPTION
**Pull request by Hugo-Inmanta**
# Description

Update python versions in the setup.py classifiers

closes #3851 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
~~- [ ] Type annotations are present~~
~~- [ ] Code is clear and sufficiently documented~~
~~- [ ] No (preventable) type errors (check using make mypy or make mypy-diff)~~
~~- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)~~
~~- [ ] Correct, in line with design~~
~~- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )~~

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

**Comment by Hugo-Inmanta** [comment]
Should I remove `python 3.5` from the classifiers as well ?

**Comment by andraskvr** [comment]
> Should I remove `python 3.5` from the classifiers as well ?

@Hugo-Inmanta Yes, I think so, if you check line 48, we already require python 3.6 at least

File: changelogs/unreleased/3851-update-classifiers-in-setup-py.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+change-type: patch
+issue-nr: 3851
+description: Update python versions in the setup.py classifiers
+destination-branches: [master]
```
Comment (edvgui) [inline changelogs/unreleased/3851-update-classifiers-in-setup-py.yml:4]: Shouldn't this go into iso5 as well?

File: changelogs/unreleased/3851-update-classifiers-in-setup-py.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+sections:
+    upgrade-note: "{{description}}"
```
Comment (edvgui) [inline changelogs/unreleased/3851-update-classifiers-in-setup-py.yml:6]: Should this really go in the changelog?  (Is this useful information to the user of the package?)

**Review by edvgui — CHANGES_REQUESTED** [review]

File: changelogs/unreleased/3851-update-classifiers-in-setup-py.yml (hunk 2 above)
Comment (Hugo-Inmanta) [reply changelogs/unreleased/3851-update-classifiers-in-setup-py.yml:6]: I guess not, I'll remove it

File: changelogs/unreleased/3851-update-classifiers-in-setup-py.yml (hunk 1 above)
Comment (Hugo-Inmanta) [reply changelogs/unreleased/3851-update-classifiers-in-setup-py.yml:4]: I wonder, is there a way for me to know towards which branches to push the changes ?

Comment (edvgui) [reply changelogs/unreleased/3851-update-classifiers-in-setup-py.yml]: I guess the documentation must mention somewhere that iso5 supports python 3.9 and iso4 python 3.6, but I could not point you to where that might be, so I can definitely not blame you for not knowing it :wink:

**Review by edvgui — APPROVED** [review]

**Pull request closed without merging**